hathor/manager.py

```diff
@@ -592,7 +615,17 @@ def _initialize_components_new(self) -> None:
         # XXX: last step before actually starting is updating the last started at timestamps
         self.tx_storage.update_last_started_at(started_at)
         self.state = self.NodeState.READY
-        self.log.info('ready')
+
+        t1 = time.time()
+        tdt = LogDuration(t1 - t0)
```

Comment:
Rename to `total_load_time`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/490/commits/a834a24942ae222f2d421b187e9a1675dc3c534d

Reply:
Thanks!